lib/maze/errors.rb

```diff
@@ -24,29 +24,29 @@ def initialize(message='Element not found', element='No element specified')
     ERROR_CODES = {
       ::Selenium::WebDriver::Error::UnknownError => {
         retry: true,
-        error_code: 10
+        error_code: Maze::Api::ExitCode::AUTOMATION_GENERIC_ERROR
       },
       ::Selenium::WebDriver::Error::WebDriverError => {
         retry: true,
-        error_code: 10
+        error_code: Maze::Api::ExitCode::AUTOMATION_GENERIC_ERROR
       },
       Maze::Error::AppiumElementNotFoundError => {
         retry: true,
-        error_code: 11
+        error_code: Maze::Api::ExitCode::AUTOMATION_ELEMENT_NOT_FOUND
       },
       ::Selenium::WebDriver::Error::NoSuchElementError => {
         retry: true,
-        error_code: 12
+        error_code: Maze::Api::ExitCode::AUTOMATION_ELEMENT_NOT_FOUND
```

Comment:
You'll notice here that the underlying exit code is the same in two case.  We could certainly have different codes for NoSuchElementError and AppiumElementNotFoundError, but I get the feeling that no one is likely to care at the exit code level - and having different codes to retry on would more likely create extra work/noise.